frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -329,9 +332,13 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
                                             <Col span={11} className="pl-4">
                                                 <RecentFeatureFlagInsights />
                                                 <div className="my-4" />
-                                                <FeatureFlagInstructions
-                                                    featureFlagKey={featureFlag.key || 'my-flag'}
-                                                />
+                                                {featureFlags[FEATURE_FLAGS.RECORDINGS_ON_FEATURE_FLAGS] ? (
```

Comment:
oh lol, no wonder I can't find the flag definition, it went into the other ux cleanup PR 😛

Reply:
oh yes lmao